handler/handler.go

```diff
@@ -96,6 +96,21 @@ func GetTaskInfo(scheduler types.Scheduler) http.HandlerFunc {
 	}
 }
 
+// ListTasks returns information about all tasks in the database.
+func ListTasks(scheduler types.Scheduler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		logrus.Debug("Fetching all tasks")
+		tasks, err := database.ListNonTerminalTasks()
+		if err != nil {
+			handleError(err, w, "Unable to fetch all tasks from the database")
+		}
+		for _, task := range tasks {
+			task.MaskedEnvironment = nil
```

Comment:
In general, I'd prefer to only mask the value and keep the variables, as it might be relevant to see what variables are being set